.claude/commands/cc/persistence/validate.md

```
@@ -0,0 +1,663 @@
+
+### Lombok
+- [✓] Entity에서 Lombok 미사용
+- [✓] Mapper에서 Lombok 미사용
+
+---
+
+## ❌ 위반 항목 (리팩토링 필요)
+
+### 1. JPA 관계 어노테이션 사용 (Zero-Tolerance)
+
+**파일**: `persistence-mysql/src/main/java/.../entity/OrderJpaEntity.java:23`
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)